### DISCUSSION TITLE
Visx leaves unwanted padding on svg element

### DISCUSSION THREAD

**Discussion by ryanbrwr** [airbnb/visx#1571] [General]
Hi there, I know this isn't a bug but looking for a way to left align or make 100% width this chart and couldn't find anything on the docs. Are there any options available to do this? Right now there is about 50px of padding in the <SVG />.

<img width="1122" alt="image" src="https://user-images.githubusercontent.com/34748418/190512933-898ce9ca-b5a9-46a1-87d9-54fde01b5b6f.png">

**Comment by williaster** [comment]
Hi @ryanbrwr 👋  thanks for checking out `visx`. This isn't an issue it's a usage question so I'm converting it to a discussion.

I assume you are using the `@visx/xychart` package, which sets a default [`margin: { top: number; right: number; bottom: number; left: number; }`](https://airbnb.io/visx/docs/xychart#XYChart_margin) on the chart. You can update that to include a `margin.left=0` if you want. it's the default to accommodate `top/right/bottom/left` axes.
